### DISCUSSION TITLE
Experiences, comments and ideas regarding the Safe Procedure.

### DISCUSSION THREAD

**Discussion by PvdAa** [asreview/asreview#1925] [Ideas]
Over the past two years, we have made extensive use of ASReview for conducting two large individual Systematic Reviews (10,603 and 31,847 studies, respectively). At the time of designing the first study protocol, the article on the Safe Procedure had not yet been published. 

Regarding the Safe Procedure, we would like to highlight several points:

1. Double Screening with Two Reviewers: This aspect, as recommended by the PRISMA method, is not sufficiently covered in the procedure. Discrepancies will inevitably arise in different phases (perfect kappa is unattainable when screening thousands of studies) during: a) screening a random set; b) screening with the initial (simple) model; and c) screening with a deep-learning model. This inevitably leads to the models of the two reviewers diverging and displaying partially different abstracts. Ultimately, when the stopping criteria are met, this results in a different total set of evaluated studies. In my experience, 1,600 studies were labeled by two reviewers in total, with about 300 studies being labeled by only one reviewer. Let's call these 'single-labeled' articles.
2. Resolving Label Discrepancies: The Safe Procedure states, “Also, it is expected two screeners will independently screen the records (…). Any disagreements should be solved before starting the next phase.” Resolving different labels is not as straightforward as it seems. Point 1 is problematic on its own because ASReview does not allow specific searches for studies while a model is being worked on. Thus, this would have to be corrected individually at the end of a phase by reviewing the studies outside of ASReview, which seems undesirable and is not mentioned in the Safe Procedure. Perhaps we missed a solution to this problem, or we might be one of the few who have double-screened all phases.
3. Inter-rater Reliability and Cohens Kappa: Reviews by two reviewers are assessed using inter-rater reliability such as Cohen's Kappa, but this is not possible when a portion of the studies is 'single-labeled'. Furthermore, it surprises me that the recommendation is for both reviewers to start with a different random set, as a kappa cannot be calculated for this. Have you considered having both reviewers start with the same random set? This would guarantee a comparable model at the start of screening. Additionally, starting with two different random sets may result in different RR_t and thus different FRR_t values. For our research, we addressed this by screening a subset of random articles 'classically' outside ASReview (we used Rayyan) and uploading the outcomes as prior knowledge into ASReview with the remaining articles unlabeled. The kappa derived from this could also be used as an argument to include/exclude 'single-labeled' articles all at once (e.g., with a very high kappa >0.90).
4. Full-Text Screening: The time savings provided by ASReview are significant, making it a promising program. However, we wondered if scaling up to full-text screening has been considered/tried. One consequence we have experienced is that facilitating the screening of large numbers of titles/abstracts makes it easier/more tempting to pose broader research questions. However, this also results in larger numbers of full-text articles needing to be read, which is ultimately a much more labor-intensive process than title/abstract screening.
5. Independent Validation and Improved Double Screening: To better map out the time savings and demonstrate independent validation of ASReview, we plan to repeat a previously conducted Systematic Review using ASReview. Perhaps this could be conducted with an updated system where double screening is better integrated. I am curious about your thoughts on this or if anyone else has done this.

**Comment by FelixWdm** [comment]
Dear @PvdAa , 

Thank you for sharing your experiences using ASReview, this is really helpful! Your points are valid and for some we're already working on making them technically possible. The SAFE Procedure is one of the several proceuders that can lead to more transparant and reproducible screening workflow, but the only one (we know of) that is published and can be referred to. But indeed, can be improved and your suggestions are really helpful.

Regarding the full-text screening; in theory this could be done by loading the text as a CSV file. Availability of these full-text is an issue and most of the testing so far was done on Title/Abstracts.

Best,
Felix

**Comment by bethgr** [comment]
Dear @PvdAa ,
Regarding point 3: I agree that both reviewers starting with a different random set would be problematic for IRR. I would recommend either starting with the same random set or having some overlap. In this case it can be treated as a missing data problem and alternative IRR measures may be helpful.

**Reply by PvdAa** [reply]
Dear @bethgr,

Yes we would recommend the same, as we used the same random set outside ASReview to ensure a comparable model to start with in ASReview.
